### DISCUSSION TITLE
Enterprise Support for ArgoCD...!

### DISCUSSION THREAD

**Discussion by Gowtham369** [argoproj/argo-cd#14211] [General]
Hello
I am not sure if this is the right place to post, still posting as it says #General:stuck_out_tongue_winking_eye:
The company I know has been recently looking for CD alternatives/options. We came across ArgoCD. and wanted to find the pricing for ArgoCD. I read that it is Open source, but I wanted to confirm if Akuity is the Enterprise Umbrella of ArgoCD...!? (Since I was able to find Pricing and Enterprise support for argoCD on their website.)
I hope someone can clear up my question...!
Thanks :raised_hands:

**Comment by crenshaw-dev** [comment]
There's Akuity, Codefresh, Harness, OpsMx, Nethopper, Red Hat OpenShift... I'm probably forgetting some. There are options.  :-)

Argo is relatively diverse. Some projects are primarily maintained by one company, often a company with a SaaS offering. Argo CD is maintained by a lot of companies and individual contributors. I believe "end user" companies actually make the majority of contributions.

**Comment by Gowtham369** [comment]
So, it is an OpenSource...(Free for any amount of services and users) That is being maintained by a few SAAS providers.
But any chance that someone can suggest the best one that would cover the shortcomings of Harness?
FYI, the use case is to use helm charts to deploy on AWS k8s. also to be able to use for some bash automation as well.

**Reply by crenshaw-dev** [reply]
> That is being maintained by a few SAAS providers

Plus a lot of end users and individual contributors.  :-) https://argo.devstats.cncf.io/d/5/companies-table?orgId=1

> any chance that someone can suggest the best one that would cover the shortcomings of Harness

I'd just hit up a few vendors for demos.

**Reply by Gowtham369** [reply]
Thanks @crenshaw-dev 
I will try to look up more info and decide which one would be best.😇
